specification/compute/resource-manager/Microsoft.Compute/stable/2022-04-04/CloudServiceRP/cloudService.json

```diff
@@ -0,0 +1,2681 @@
+{
```

Comment:
Is everything in this file new? There's still no previous version to compare to.

Reply:
previous version was 2021-03-01. 
The changes for this version is all in this commit https://github.com/Azure/azure-rest-api-specs/pull/19468/commits/ed2c211f02777ea940b14465c43acd45e6ea8de1

Reply:
@zizw123